atd-vzd/triggers/is_crash_in_austin_full_purpose.sql

```diff
@@ -0,0 +1,13 @@
+-- This function determines whether crash coordinates fall within the Austin Full Purpose jurisdiction polygon.
+-- If the crash has no coordinates, it returns whether the crash is in Austin (city_id = 5)
+CREATE OR REPLACE FUNCTION is_crash_in_austin_full_purpose(location geometry(Geometry,4326), city_id int) RETURNS boolean AS $$
+        BEGIN
+            -- If we dont have coordinates for the crash and the city_id is Austin (22), return true
+            IF (location IS NULL) THEN --TODO: Do we need to handle if city_id is null? If so should that make the afp flag default to false?
```

Comment:
do crashes ever have `city_id = null`? if so, how should we handle that here? no position and no city id means not in austin full purpose? i can see in the DB `city_id` is nullable

Reply:
I don't see any nulls in the prod DB - just some marked as `9999` which doesn't correspond to a value in the lookup table. Maybe the PR for the other part of this Lambda will uncover some more info?